customTests/azure_ib_write_bw_non_gdr.nhc

```diff
@@ -77,7 +77,7 @@ function ib_write(){
          return 1
    fi
    dbg "ib_write_lb_${device}: $ib_bandwidth Gbps"
-   pass 0 "$FUNCNAME: IB write bandwidth non gdr test passed"
+   pass 0 "check_ib_bw_non_gdr: IB write bandwidth non gdr test passed for IB=$device, IB BW=$ib_bandwidth Gbps"
```

Comment:
Same as my other comment related to die message.

Reply:
Addressed